packages/big-design/src/components/Select/types.ts

```diff
@@ -0,0 +1,31 @@
+import { Placement } from 'popper.js';
+
+import { ListItemProps } from '../List/Item';
+
+export interface SelectProps extends Omit<React.HTMLAttributes<HTMLUListElement>, 'children' | 'onChange'> {
+  action?: Action;
+  disabled?: boolean;
+  error?: string;
+  label?: string;
+  maxHeight?: number;
+  multi?: boolean;
+  options: Option[];
+  placement?: Placement;
+  positionFixed?: boolean;
+  required?: boolean;
+  value?: string | number | Array<string | number>;
+  onChange(value: string | number | Array<string | number>): void;
+}
+
+interface BaseItem extends Omit<ListItemProps, 'children' | 'content' | 'value'> {
+  content: string;
```

Comment:
Just wondering whether we should allow `content` to take in component as well? One of the use case is 
<img width="445" alt="Screen Shot 2019-10-24 at 10 41 25 am" src="https://user-images.githubusercontent.com/22089936/67442009-db5f9d00-f64a-11e9-9032-6fa13debbc38.png">

Reply:
@RobinOng What do you think instead of accepting a `component` we add another prop `subContent` / `subText` or something like that, to keep consistency as much as possible. If we end up getting a ton of different designs we can fallback to `component` in the future.

Reply:
The `subContent` / `subText` approach sounds much better 👍